src/Storage/Device/S3.php

```diff
@@ -121,9 +214,10 @@ public function delete(string $path, bool $recursive = false):bool
      *
      * @return int
      */
-    public function getFileSize(string $path):int
+    public function getFileSize(string $path): int
     {
-        return 0;
+        $res = $this->getInfo($path);
+        return $res['size'];
```

Comment:
It's a best practice to validate the key actually exist. Something like: `return (isset($res['size'])) ? $res['size'] : 0;` or the shorter syntax: `return $res['size'] ?? 0;`